wordpress-mobile/WordPress-iOS
# Likes Notifications:  add core data entities for user details.

Ref #15662 

This adds two new entities to store Likes data, and initial classes for each.
- `LikeUser` - for the user info.
- `LikeUserPreferredBlog` - for the user's preferred blog.

Nothing in the app is actually using them yet.

To test:
- Verify the app builds and runs. 
- Verify it is using version 122 of the data model and the new entities are present.
- Check `LikeUser` attributes to be sure they match up with WPKit `RemoteLikeUser` [properties](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/develop/WordPressKit/RemoteUser%2BLikes.swift#L3). 
- Check `LikeUserPreferredBlog` attributes to be sure they match up with WPKit `RemoteLikeUserPreferredBlog` [properties](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/develop/WordPressKit/RemoteUser%2BLikes.swift#L29). 
- Verify the relationships between `LikeUser` and `LikeUserPreferredBlog` is 1-to-1, optional from `LikeUser` to `LikeUserPreferredBlog`.


## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
